src-docs/src/views/tour/guidelines.js

```diff
@@ -0,0 +1,186 @@
+      <Link to="/display/empty-prompt">
+        <strong>EuiEmptyPrompt</strong>
+      </Link>
+      .
+    </p>
+    <p>
+      For certain users, product tours can feel intrusive so first assess the
+      fit for your use case and users. The goal is for the product tour to be a
+      tool that helps the user learn new things and accomplish their goals.
+      Three good scenarios for using a product tour are:
+    </p>
+  </EuiText>
+);
+
+export default () => (
+  <>
+    <EuiText grow={false}>
+      <p>
+        This page documents best practices for tour design including content,
+        length and use cases.
+      </p>
+    </EuiText>
+
+    <GuideRule heading="When to use tours" description={whenDescription}>
+      <EuiFlexItem>
+        <EuiText grow={false}>
+          <ol>
+            <li>New users seeing an interface for the first time</li>
+            <li>Novice users trying to gain proficiency in your application</li>
+            <li>
+              Existing users need to be onboarded when new features or redesigns
+              are released
+            </li>
+          </ol>
+        </EuiText>
+        <EuiSpacer />
+        <EuiText grow={false}>
+          Additionally, consider asking users if they’re interested in checking
+          out your product tour instead of just showing it to them.
+        </EuiText>
+      </EuiFlexItem>
+    </GuideRule>
+
+    <GuideRule
+      heading="Provide concise yet valuable information"
+      description="If you include information that is too obvious or basic, it is more likely that the user will dismiss the product tour and start perceiving them as low value. If further explanation is needed, consider linking out to documentation."
+    >
+      <GuideRuleExample
+        type="do"
+        text="Keep the content of each step short while making sure to provide useful information."
+      >
+        <EuiImage
+          alt="concise content in tour step"
+          url={image1Do}
+          height="252"
+        />
+      </GuideRuleExample>
+
+      <GuideRuleExample
+        type="dont"
+        text="Use lenghty text that contains a lot of detail. Instead you can add a link for users to learn more."
+      >
+        <EuiImage
+          alt="lengthy content in tour step"
+          url={image1Dont}
+          height="252"
+        />
+      </GuideRuleExample>
+    </GuideRule>
+
+    <GuideRule
+      heading="Explain why the actions you want users to perform are useful"
+      description="If users see value in an action they’ll be more likely to engage."
+    >
+      <EuiFlexItem>
+        <EuiPanel
+          color="subdued"
+          paddingSize="l"
+          hasShadow={false}
+          style={{ justifyContent: 'center', display: 'flex' }}
+        >
+          <EuiPanel style={{ maxWidth: 520 }} paddingSize="s">
+            <EuiImage alt="tour useful step" url={image2Example} />
+          </EuiPanel>
+        </EuiPanel>
```

Comment:
The code looks like:

```suggestion
      <EuiAspectRatio width={2} height={1}>
        <iframe
          title="tour useful step"
          width="800"
          height="450"
          src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FOTA9yTu1X0gIiJCthgUtPD%2F%255BGuidelines%255D-EuiTour%3Fnode-id%3D152%253A9193"
        />
     </EuiAspectRatio>
```

Reply:
The `EuiAspectRatio` wrapper makes sure it shrinks on smaller screens.

Reply:
Yeah I'd move all your final frames to the Tour page in the Elastic Library. I'd make sure your frames don't have a background and then when you copy/paste the embed code, you'll want to remove the `style` tag that comes with it.